pipeline/extract/analytics.py

```diff
@@ -0,0 +1,20 @@
+from typing import List
+
+from prefect import task
+
+from pipeline.types import AnalyticsEvent
+
+
+@task
+def extract_analytics(db) -> List[AnalyticsEvent]:
+    ref = db.reference(f"analytics/")
```

Comment:
No f-string needed here, nor the slash at the end.

Reply:
Woops! Thanks for catching that.